### PR TITLE
Final Milestone: Rails Form

### PR DESCRIPTION
**Pull request by AhmedTohamy01**
Final Milestone: Rails Form Project, this includes:

- Add a from to create a new user.
- Add a form to edit existing user.

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
```
Comment (maelfosso) [inline app/views/users/new.html.erb:2]: Remove that, please.

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+<!--
+<form action="/users" method="post" accept-charset="UTF-8">
```
Comment (maelfosso) [inline app/views/users/new.html.erb:4]: it's better to use **built-in named route helper methods** like **users_path** that `/users`. Re-read this please https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+  <label for="email">Email:</label>
+  <input type="text" id="email" name="user[email]"><br><br>
+
+  <input type="submit" value="Submit">
+</form>
+-->
+
+<!--
+<%= form_tag("/users", method: "post") do %>
```
Comment (maelfosso) [inline app/views/users/new.html.erb:21]: it's better to use **built-in named route helper methods** like **users_path** that `/users`. Re-read this please https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing

**Comment by AhmedTohamy01** [comment]
Hi Mael,

Thank you for your comments, We did all required changes.

Best Regards

**Comment by uryelah** [comment]
Hi **Sunday Ezeilo & Ahmed Tohamy**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job working on the pointed issues!

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**


---------

    
    
**Keep being awesome devs and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by AhmedTohamy01** [comment]
Thank you Sarah, Much Appreciated :)

**Pull request merged**